util/password/password.go

```diff
@@ -49,7 +54,7 @@ func Hash(password []byte) (hash []byte, err error) {
 		return
 	}
 
-	return bcrypt.GenerateFromPassword(password, 0)
+	return bcrypt.GenerateFromPassword(password, Cost)
```

Comment:
I dislike this change. My intention on passing 0 here is to always use a reasonable default provided by the maintainer(s) of `golang.org/x/crypto/bcrypt`.